jenkins/build-pr.sh

```diff
@@ -37,7 +33,7 @@ do
   fi
 done
 
-echo "Building with ert=$ERT_REVISION opm-common=$OPM_COMMON_REVISION opm-parser=${upstreamRev[opm-parser]} opm-material=${upstreamRev[opm-material]} opm-core=${upstreamRev[opm-core]} opm-grid=${upstreamRev[opm-grid]} opm-output=${upstreamRev[opm-output]} opm-upscaling=$sha1"
+echo "Building with opm-common=$OPM_COMMON_REVISION ert=${upstreamRev[ert]} opm-parser=${upstreamRev[opm-parser]} opm-material=${upstreamRev[opm-material]} opm-core=${upstreamRev[opm-core]} opm-grid=${upstreamRev[opm-grid]} opm-output=${upstreamRev[opm-output]} opm-upscaling=$sha1"
```

Comment:
Tangential note: Do we want/need this on a single line or might we consider multiple lines like
```sh
cat<<EOM
Building with
  opm-common    = ${OPM_COMMON_REVISION}
  ert           = ${upstreamRev[ert]}
  opm-parser    = ${upstreamRev[opm-parser]}
  opm-material  = ${upstreamRev[opm-material]}
  opm-core      = ${upstreamRev[opm-core]}
  opm-grid      = ${upstreamRev[opm-grid]}
  opm-output    = ${upstreamRev[opm-output]}
  opm-upscaling = $sha1
EOM
```